src/Razor/Microsoft.AspNetCore.Razor.Language/src/DefaultRazorProjectItem.cs

```diff
@@ -17,13 +17,15 @@ internal class DefaultRazorProjectItem : RazorProjectItem
         /// <param name="filePath">The path.</param>
         /// <param name="fileKind">The file kind. If null, the document kind will be inferred from the file extension.</param>
         /// <param name="file">The <see cref="FileInfo"/>.</param>
-        public DefaultRazorProjectItem(string basePath, string filePath, string relativePhysicalPath, string fileKind, FileInfo file)
+        /// <param name="cssScope">A scope identifier that will be used on elements in the generated class, or null.</param>
+        public DefaultRazorProjectItem(string basePath, string filePath, string relativePhysicalPath, string fileKind, FileInfo file, string cssScope)
```

Comment:
Can we do `string cssScope = null` here to make this backwards compatible?

Reply:
We could, but the class is internal so hopefully we don’t need to be concerned with back-compat.